compiler/damlc/daml-ide-core/test/Development/IDE/State/API/Testing.hs

```diff
@@ -551,7 +552,7 @@ subgraphToExpectedSubgraph vSubgraph = ExpectedSubGraph vNodes vFields vTplName
 graphToExpectedGraph :: V.Graph -> ExpectedGraph
 graphToExpectedGraph vGraph = ExpectedGraph vSubgrpaghs vEdges
     where vSubgrpaghs = map subgraphToExpectedSubgraph (V.subgraphs vGraph)
-          vEdges = map (\(c1,c2) -> (expectedChcDetails c1, expectedChcDetails c2)) (V.edges vGraph)
+          vEdges = map (bimap expectedChcDetails expectedChcDetails) (V.edges vGraph)
```

Comment:
We end up with a slightly different version of HLint that yells about these hints.